.eslintcache

```diff
@@ -0,0 +1 @@
0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"42","messages":"43","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"44","messages":"45","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"46","messages":"47","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"48","messages":"49","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"50","messages":"51","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"52","messages":"53","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"54","messages":"55","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"56","messages":"57","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"58","messages":"59","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"60","messages":"61","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"62","messages":"63","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"64","messages":"65","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"66","messages":"67","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"68","messages":"69","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/Michelle/Documents/GitHub/portfolio/src/index.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/App.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/reportWebVitals.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/nav_bar_component.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/footer.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/home.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/resume.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/projects.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/rain.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/bookshelf.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/house.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/fishtank.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/magic8.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/animal.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/city.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/spork.js",[],"/Users/Michelle/Documents/GitHub/portfolio/src/components/p5js-components/plate.js",[]]
```

Comment:
add this file to your gitignore (but research to make sure that's the right move first)

Reply:
also delete it from your git cache to git rid of it here (google how to do this)